apps/src/codebridge/codebridgeContext/codebridgeContextSourceReducer.ts

```diff
@@ -78,7 +78,7 @@ export const sourceReducer = (
         action.payload
       );
 
-      if (source.files[fileId].contents === contents) {
+      if (source.files[fileId]?.contents === contents) {
```

Comment:
I starting getting an error at this line when navigating between a widget view-enabled level to a non-widget view-enabled level so check for existence of `files[fileId]` before trying to access its `contents`.